types/chain.go

```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"errors"
+	"github.com/trustwallet/go-primitives/coin"
```

Comment:
Can you split packages? Separate group for built-in package, separate group for project packages and separate group for third-party packages.

For example:
```
import (
        "errors"
        "strings"

        "github.com/trustwallet/go-primitives/coin"

        "some/external/package/here"
        ""
)
```

Reply:
Sorry, don't understand what is built-in packages in your case? should I move it to an already existed package or create a new "chain"?

Reply:
Just use `goimports` in your IDE